src/main/java/frc/robot/commands/autonomousCommands/TrajectoryTutCommand.java

```diff
@@ -0,0 +1,104 @@
+package frc.robot.commands.autonomousCommands;
+
+import java.util.List;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.controller.RamseteController;
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.trajectory.Trajectory;
+import edu.wpi.first.math.trajectory.TrajectoryConfig;
+import edu.wpi.first.math.trajectory.TrajectoryGenerator;
+import edu.wpi.first.math.trajectory.constraint.DifferentialDriveVoltageConstraint;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.RamseteCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.constants.DriveConstants;
+import frc.robot.subsystems.drive.Drive;
+
+public class TrajectoryTutCommand extends SequentialCommandGroup {
```

Comment:
Let's make this class name and file name called `TrajectoryTutorialCommandGroup`